src/server9.py

```diff
@@ -60,7 +60,7 @@ def show_comments():
     for entry in ENTRIES:
         out += "<p>" + entry + "</p>"
     out += "<link rel=stylesheet src=/comment.css>"
-    out += "<div id=errors></div>"
+    out += "<errors></errors>"
```

Comment:
Why use a non-standard element here? Also doesn't this now not match the text?

Reply:
Well, we have to do it base don a tag, because that's the only selector we have. Which one do you want?

https://way2tutorial.com/html/tag/index.php

I could do `<span>` just because it doesn't exist elsewhere, or `<pre>`?

Reply:
Oh man, I'm a moron, I forgot about this. I think technically `<label>` is recommended, since it describes an input element.

Reply:
```suggestion
    out += "<label></label>"
```